.github/workflows/main.yml

```diff
@@ -12,6 +12,8 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
+      - name: Refresh dependencies
+        run: apt update
```

Comment:
sudo?

Reply:
oh yeah I didn't realize it wasn't running as root.  Will add.